test-watch-in-docker.sh.protocols.yaml

```diff
@@ -0,0 +1 @@
+- /usr/bin/docker run --init --rm -it --cap-add=SYS_PTRACE --entrypoint /bin/bash -v /root/project:/root/ check-protocols -c cargo watch -x test
```

Comment:
the tests fail because of the way quotes are handled between script <-> protocol. i'm not positive that this isn't my own fault, however... running locally:

```sh
# script (with quotes included)
docker run --init --rm -it --cap-add=SYS_PTRACE --entrypoint "/bin/bash" -v $(pwd):/root/ check-protocols -c "cargo watch -x test"
# protocol (with quotes included at the end)
- /usr/bin/docker run --init --rm -it --cap-add=SYS_PTRACE --entrypoint /bin/bash -v /root:/root/ check-protocols -c "cargo watch -x test"
```
i get
```
error:
  expected: /usr/bin/docker run --init --rm -it --cap-add=SYS_PTRACE --entrypoint /bin/bash -v /root:/root/ check-protocols -c "cargo watch -x test"
  received: /usr/bin/docker run --init --rm -it --cap-add=SYS_PTRACE --entrypoint /bin/bash -v /root:/root/ check-protocols -c cargo watch -x test
```
`received` makes me think that i should remove the quotes in my protocol, but if i do that, i get
```
error:
  expected: /usr/bin/docker run --init --rm -it --cap-add=SYS_PTRACE --entrypoint /bin/bash -v /root:/root/ check-protocols -c cargo watch -x test
  received: /usr/bin/docker run --init --rm -it --cap-add=SYS_PTRACE --entrypoint /bin/bash -v /root:/root/ check-protocols -c cargo watch -x test
```
(they are the same string, i double checked that there were no hidden characters)

...i dug into this, it looks like the `arguments` here https://github.com/Originate/check-protocols/blob/master/src/protocol/mod.rs#L63 don't match (possibly something that should be added to the formatted error output?)

```
# self.arguments
["run", "--init", "--rm", "-it", "--cap-add=SYS_PTRACE", "--entrypoint", "/bin/bash", "-v", "/root:/root/", "check-protocols", "-c", "cargo", "watch", "-x", "test"]
# arguments
["run", "--init", "--rm", "-it", "--cap-add=SYS_PTRACE", "--entrypoint", "/bin/bash", "-v", "/root:/root/", "check-protocols", "-c", "cargo watch -x test"]
```

Reply:
Yeah, quoted arguments don't really work right now. There's this ticket for that: #67.

Reply:
I think this shouldn't block this PR though. This PR would also be good without the test protocol. As much as I like using the tool itself in this repo, this protocol is also not *that* useful, since it just repeats the command from the script.

Reply:
agreed, they aren't very helpful for one liners. i'll remove this.